docs/operating-eck/troubleshooting/common-problems.asciidoc

```diff
@@ -84,3 +84,73 @@ type: Opaque
 ----
 
 Failure to do so can cause data loss.
+
+[id="{p}-{page_id}-pod-updates"]
+== Pods are not replaced after a configuration update
+
+The update of an existing Elasticsearch cluster configuration can fail because the operator is unable to apply the changes required while replacing the pods of a given Elasticsearch cluster.
+
+This can be identified by the `AGE` of the pod being older than the configuration update, or the `STATUS` of the pods:
+
+[source,sh]
+----
+kubectl get pods
+
+NAME                      READY   STATUS    RESTARTS   AGE
+quickstart-es-default-0   1/1     Running   0          15m
+quickstart-es-default-1   1/1     Running   0          15m
+quickstart-es-default-2   1/1     Running   0          15m
+----
+
+Possible causes include:
+
+* The Elasticsearch cluster is not healthy.
++
+[source,sh]
+----
+kubectl get elasticsearch
+
+NAME         HEALTH   NODES   VERSION   PHASE   AGE
+TODO: Finish example
```

Comment:
- [ ] Need output example of a yellow cluster

Reply:
Hi,

If it helps a cluster with unassigned shards would look like this:

```
NAME                                                              HEALTH   NODES   VERSION   PHASE   AGE
elasticsearch.elasticsearch.k8s.elastic.co/elasticsearch-sample   yellow   1       7.9.2     Ready   3m50s
```

Not sure if we should use `{version}` as it might break formatting.

Reply:
I added the example provided by @barkbay.
@a03nikki is this what you expected?

Reply:
@alaudazzi : Yes, this is what I was expecting for the TODO.